csaf/src/verification/check/mod.rs

```diff
@@ -43,6 +46,195 @@ impl Checking {
     }
 }
 
+pub fn check_vulnerabilities_size(csaf: &Csaf) -> Vec<CheckError> {
+    let result;
+    if let Some(vs) = &csaf.vulnerabilities {
+        result = vs.len() > 0;
```

Comment:
I would use `is_empty`, which makes this clearer. This might also be flagged by `clippy`.